include/config/config_goddard.h

```diff
@@ -7,7 +7,7 @@
 /**
  * Enables Goddard (the Mario head).
  */
-// #define KEEP_MARIO_HEAD
+#define KEEP_MARIO_HEAD
```

Comment:
whys this being toggled?

Reply:
Temporarily for testing the rumble pak graphic. I'll re-disable it once this is ready to merge.